README.md

```diff
@@ -61,20 +87,18 @@ Optionally, you can download and install [Postman](https://www.getpostman.com) t
 ### Getting Started
 Service interactions can be deployed, tested and developed locally using Docker Compose as described below. For further instructions & information about __i40-aas__ check [the documentation](docs/README.md)
 
-Run __i40-aas__ locally using any of these options:
```

Comment:
looks like a wrong local merge. This description should not be deleted, as it is the new one.